src/polygon/EtherOrcsItems.sol

```diff
@@ -81,6 +85,16 @@ contract EtherOrcsItems is IERC1155 {
         _burn(from, id, value);
     }
 
+    function wrapBoneshard(uint256 value) external {
+        ERC20Like(boneshards).transferFrom(msg.sender, address(this), value);
+        _mint(msg.sender, WRAPPED_BONESHARD_ID, value);
+    }
+
+    function unwrapBoneshard(uint256 value) external {
+        _burn(msg.sender, WRAPPED_BONESHARD_ID, value);
+        ERC20Like(boneshards).transferFrom(address(this), msg.sender, value);
+    }
+
```

Comment:
This is good.

One thing that might be an extra layer of proptection, if desired, is to add a check to make sure that both the wrap and the unwrap are dealing with whole amounts of bs.